pxtblocks/blocklyloader.ts

```diff
@@ -913,21 +1058,168 @@ namespace pxt.blocks {
             return xmlList;
         };
 
+        // builtin variables_get
+        let msg:any = Blockly.Msg;
+        msg.VARIABLES_GET_CREATE_SET = lf("Create 'set %1'");
+        installHelpResources(
+            'variables_get',
+            lf("get the value of a variable"),
+            lf("Returns the value of this variable."),
+            '/blocks/variables'
+        );
+
+        // builtin variables_set
+        msg.VARIABLES_SET = lf("set %1 to %2");
+        msg.VARIABLES_DEFAULT_NAME = lf("_default_var");
+        //XXX Do not translate the default variable name.
+        //XXX Variable names with Unicode character are harmful at this point.
+        msg.VARIABLES_SET_CREATE_GET = lf("Create 'get %1'");
+        installHelpResources(
+            'variables_set',
+            lf("assign the value of a variable"),
+            lf("Sets this variable to be equal to the input."),
+            '/blocks/variables/assign'
+        );
+
+        // pxt variables_change
         Blockly.Blocks['variables_change'] = {
             init: function () {
-                this.appendDummyInput()
-                    .appendField("change")
-                    .appendField(new Blockly.FieldVariable("item"), "VAR");
-                this.appendValueInput("VALUE")
-                    .setCheck("Number")
-                    .appendField("by");
-                this.setInputsInline(true);
-                this.setPreviousStatement(true);
-                this.setNextStatement(true);
-                this.setTooltip(lf("Changes the value of the variable by this amount"));
-                this.setHelpUrl('/reference/assign');
-                this.setColour(blockColors['variables']);
+                this.jsonInit({
+                  "message0": lf("change %1 by %2"),
+                  "args0": [
+                    {
+                      "type": "field_variable",
+                      "name": "VAR",
+                      "variable": lf("_default_var")
```

Comment:
Same as ``_for_do`` and ``_while_do``, please open an issue and we'll fix this in a future commit.